### PR TITLE
add size limit to metadata of room & participant

### PR DESCRIPTION
**Pull request by cnderrauber**

### CODE REVIEW

File: pkg/service/errors.go [hunk 1]
```diff
@@ -12,4 +12,5 @@ var (
 	ErrTrackNotFound        = errors.New("track is not found")
 	ErrWebHookMissingAPIKey = errors.New("api_key is required to use webhooks")
 	ErrOperationFailed      = errors.New("operation cannot be completed")
+	ErrMetadataExceedLimits = errors.New("metadata size exceed limits")
```
Comment (boks1971) [inline pkg/service/errors.go:15]: :nit:
```
ErrMetadataExceedsLimit = errors.New("metadata size exceeds limit")
```

File: pkg/service/roomservice_test.go [hunk 2]
```diff
@@ -42,11 +44,62 @@ func TestDeleteRoom(t *testing.T) {
 	})
 }
 
+func TestMetaDataLimits(t *testing.T) {
+	t.Run("metadata exceed limits", func(t *testing.T) {
+		svc := newTestRoomService()
```
Comment (boks1971) [inline pkg/service/roomservice_test.go:49]: Maybe worth passing in the max metadata size here and check with 0 value also to make sure that no limit works when it is set to 0.

**Review by boks1971 — APPROVED** [review]

File: pkg/service/errors.go (hunk 1 above)
Comment (cnderrauber) [reply pkg/service/errors.go]: Done

File: pkg/service/roomservice_test.go (hunk 2 above)
Comment (cnderrauber) [reply pkg/service/roomservice_test.go]: Done

**Pull request merged**